src/amo/components/AddonMoreInfo/index.js

```diff
@@ -54,6 +78,38 @@ export class AddonMoreInfoBase extends React.Component<InternalProps> {
       });
     }
 
+    const addonType = addon.type;
+
+    let categories = [];
+
+    const reledCategories = [];
+
+    if (
+      categoriesState &&
+      categoriesState[clientApp] &&
+      categoriesState[clientApp][addonType]
+    ) {
+      categories = Object.keys(categoriesState[clientApp][addonType]).map(
+        (key) => categoriesState[clientApp][addonType][key],
+      );
+
+      categories.forEach((r) => {
+        if (addon.categories && addon.categories[clientApp].includes(r.slug)) {
+          reledCategories.push(r.name);
+        }
+      });
+    }
+
+    if (!errorHandler.hasError() && !loading && !categories.length) {
+      return (
```

Comment:
I don't think you want to just return a `Card` here. Wouldn't that mean that none of the other data would be displayed for the add-on? I suspect that you don't need any special logic for this here at all.

Reply:
Yes, you are correct, if we add this than whole addon will not displayed.